java-core/solid/fruit-shop-tests.md

```diff
@@ -1,6 +1,7 @@
 ## Common mistakes (jv-fruit-shop-tests)
 
-##### Don't begin class or method implementation with empty line.  Remove all redundant empty lines, be careful :)
+#### Don't begin class or method implementation with empty line. 
+Remove all redundant empty lines, be careful :)
```

Comment:
sometimes we use one line for these statements and sometimes two lines
let's use one style, but change it in the next PR